contrib/functions/go/annotate-apply-time-mutations/README.md

```diff
@@ -30,36 +30,137 @@ pipeline:
     - image: gcr.io/kpt-fn-contrib/annotate-apply-time-mutations:unstable
 ```
 
-or imperatively like:
+The function can also be executed imperatively:
 
 ```shell
 kpt fn eval --image gcr.io/kpt-fn-contrib/annotate-apply-time-mutations:unstable
 ```
 
+The function will generate and write the
+`config.kubernetes.io/apply-time-mutation` annotation on the target object, the
+object with the apply-time-mutation comment on one or more of its fields.
+
+The function does not perform the mutation on the target object itself. The
+mutation is performed by `kpt live apply`, which reads the annotation as input.
+So the function should be run by the user before applying.
+
+## Inline Field Comment Input
+
+Inline field comments can be used as an alternate way to specify mutations. This
+function will convert `apply-time-mutation` comments into apply-time-mutation
+annotations.
+
+With inline comments, the mutation is specified closer to the target field that
+will be updated. This can aid debugging and onboarding by reducing indireciton.
+It also reduces the configuration required, because the target object and field
+don't need to be explicitly specified.
+
+Inline field comments can be specified with the following format:
+
+```
+# apply-time-mutation: [prefix]${[group]/[version]/namespaces/[source-namespace]/[kind]/[source-name]:[source-field-path]}[suffix]
+```
 
-The `annotate-apply-time-mutations` function does the following:
+`prefix` and `suffix` are optional. They are constant strings that surround dynamic
+content from the source field. When specified, the function will copy them into
+the field itself, surrounding a generated token.
 
-1.  Scans the package for `apply-time-mutation` comment markup.
-2.  Appends the equivalent `config.k8s.io/apply-time-mutation` annotation to the same.
+```
+field: [prefix]$ref[suffix] # apply-time-mutation: ...
+```
+
+`version` is also optional. It's generally recommended to avoid specifying the
```

Comment:
When `version` is omitted, should I use remove one `/`. IOW, `[group]//namespaces/[source-namespace]/[kind]/[source-name]` or `[group]/namespaces/[source-namespace]/[kind]/[source-name]`.
Please clarify.